src/main/cpp/sdk3/ImageClsSDK3.h

```diff
@@ -49,7 +49,7 @@ class ImageSDK3
         *iAuxOutSource; // 0-FireRow1; 1-FireRowN; 3-FireAll; 4-FireAny
 
     // control flow
-    boolean isStopPressed;
+    bool isStopPressed;
```

Comment:
@danielaik Is sdk3 still used in this project? I don't think this code was compiling before because of this. If it's not used anymore we should delete the folder.
Otherwise, it now compile with CMake and produce the dll 😃